rpm/generic/zfs.spec.in

```diff
@@ -70,6 +60,29 @@
 %define _systemd 1
 %endif
 
+# When not specified default to distribution provided version.  This
+# is normally Python 3, but for RHEL <= 7 only Python 2 is provided.
+%if %{undefined __use_python}
+%if 0%{?rhel} && 0%{?rhel} <= 7
+%define	__python              python2
+%define	__python_pkg_version  2
+%else
+%define	__python              python3
+%define	__python_pkg_version  3
```

Comment:
This needs to be `/usr/bin/pythonX`, I believe.

Examples: https://src.fedoraproject.org/rpms/python-rpm-macros/blob/master/f/macros.python-srpm#_1-6

Reply:
Yup, fixed.